modules/sharedaddy/sharing.css

```diff
@@ -190,8 +213,19 @@ body .sd-content ul li.share-custom.no-icon a span {
 
 .sd-social-icon .sd-content ul li a.share-more {
 	position: relative;
-	top: 2px;
+	top: -4px;
 }
+@media screen and (-webkit-min-device-pixel-ratio:0) {
+	.sd-social-icon .sd-content ul li a.share-more {
+		top: 2px;
+	}
+}
+@-moz-document url-prefix() {
```

Comment:
I'm not used to this CSS. It would be nice to have a comment here explaining what this is for.